packages/template-retail-react-app/app/components/search/index.jsx

```diff
@@ -85,6 +92,9 @@ const formatSuggestions = (searchSuggestions, input) => {
  * @return  {React.ReactElement} - SearchInput component
  */
 const Search = (props) => {
+    const config = getConfig()
+    const {enabled, askAgentOnSearch} = JSON.parse(config.app.commerceAgent)
```

Comment:
Is is an example of how maintaining this parameter as an object vs as a string will simplify the code because we won't have these JSON.parse calls scattered around?